Source/Scene/GltfLoader.js

```diff
@@ -1175,6 +1203,7 @@ function loadTexture(
 function loadMaterial(loader, gltf, gltfMaterial, supportedImageFormats) {
   const material = new Material();
 
+  // TODO: don't make it load material if it's for classification
```

Comment:
I think when I wrote this comment I meant specific components of the material -- normal maps, for example, aren't needed because normals aren't used for classification. I'm also not sure what effect other PBR parameters will have on its appearance, but I suppose we could just keep them in and see what happens.

Reply:
And actually, old [`ClassificationModel`](https://github.com/CesiumGS/cesium/blob/679453d2012c203faeca010debfd4a8ffa858b40/Source/Scene/ClassificationModel.js) only used the `POSITION` and `BATCH_ID` attributes of the model. So frankly you couldn't have materials with textures because the UVs were never accounted for.